tests/core/tools/color.js

```diff
@@ -0,0 +1,264 @@
+/* bender-tags: editor */
+
+
+/**
+ * Basic definitions
+ * HEX 	 -> valid HEX Color Value with # value: #FFF or #FFFFFF
+ * 3-HEX -> HEX but with only 3 characters value: #FFF
+ * 6-HEX -> HEX but with exactly 6 characters value: #FFFFFF
+ * 8-HEX -> HEX but with exactly 8 characters value: #FFFFFF00. Last two characters are for alpha
+ */
+( function() {
+	'use strict';
+	bender.test( {
+		setUp: function() {
+			this.createColor = function( colorCode ) {
```

Comment:
Honestly, I made `createColor` only for tests just to avoid typing `CKEDITOR.tools.color` everywhere. In case something change in `color` class, or namespace - there is only one place where changes need to be done.

Static creator vs `new`...
- there are approaches that you should never create any class with `new` keyword. Besides factory methods, which are implemented inside a class. So they are allowed to know the complex algorithm of creating this object. So whenever big preparations are needed before object creation, we would rather encapsulate this logic inside a class (with the static creator).
This helps avoid huge code changes in case of object construction change.  But I think it's more useful when you need to create many dependencies for object creation.
For such a plain object like ours `color`, one thing that could change is the parameters list - so it would require modifying the static creator function declaration anyway. No befits for me here.
- there was one moment when I think about it. We could have base color class and static factory method. Based on input it could return a specific color object. So we would have `ColorRgb`, `ColorHsl`, `ColorHex` - all of them extend `color`. Each of these classes would contain a parser just for its type. However, there would be many more combinations with conversion between colors objects. 
Probably there should be another class: `converter` which would take different color objects and then convert them? So anyway we would need to extract some basic logic from each class for such conversion. I think this approach does not suit me. Where parsers in separate classes would make more clean code, then `converter` would make it messier.